ush/config_defaults.yaml

```diff
@@ -1466,6 +1488,26 @@ task_get_extrn_lbcs:
   LBC_SPEC_INTVL_HRS: 6
   EXTRN_MDL_LBCS_OFFSET_HRS: '{{ 3 if EXTRN_MDL_NAME_LBCS == "RAP" else 0 }}'
   FV3GFS_FILE_FMT_LBCS: "nemsio"
+  #
+  #-----------------------------------------------------------------------
+  #
+  # LBCS_SEARCH_HRS:
+  #  When search boundary conditions tasks from previous cycles in prep_cyc step,
+  #  For example: 0 means search start for the same cycle lbcs task.
+  #               1 means search start for 1-h previous cycle lbcs task.
+  #               2 means search start for 2-h previous cycle lbcs task.
+  #
+  # EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS:
+  #  When search boundary conditions from previous cycles in prep_start step,
+  #  the search will start at cycle before (this parameter) of current cycle.
+  #  For example: 0 means search start at the same cycle lbcs directory.
+  #               1 means search start at 1-h previous cycle  lbcs directory.
+  #               2 means search start at 2-h previous cycle  lbcs directory.
+  #
+  #-----------------------------------------------------------------------
+  #
+  LBCS_SEARCH_HRS: 6
+  EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS: 0
```

Comment:
As far as I can tell, the functional part that uses these variables is not included with this PR. It might be good to either include that, or remove these variables.

Reply:
`LBCS_SEARCH_HRS` is used only in the workflow:
```
           <taskdep task="&TN_MAKE_ICS;{{ uscore_ensmem_name }}"/>
           <or>
             {%- for h in range(0,lbcs_search_hrs) %} 
             <metataskdep metatask="&TN_MAKE_LBCS;{{ uscore_ensmem_name }}" cycle_offset="-{{ h }}:00:00"/>
             {%- endfor %}
           </or>
```
The other one is used in task `prepstart` ex-script, but since it relates to LBCS, I've put it here.